src/components/Course/Content/ExerciseContent/index.js

```diff
@@ -114,6 +114,28 @@ const RenderContent = ({ data }) => {
     }
   }
   if (data.component === "table") {
+    // return tableData(data);
+    //Changing list data from row to column
+    const allData = data.value.map((item) => item.items);
+    const dataInRow = [];
+    for (const i in allData[0]) {
+      for (const j in allData) {
+        dataInRow.push(allData[j][i]);
+      }
+    }
+    const dataInCol = [];
+    for (const i in allData[0]) {
+      const temArray = [];
+      for (const j in dataInRow) {
+        if (j == 3) {
+          break;
+        } else {
+          temArray.push(dataInRow[j]);
+        }
+      }
```

Comment:
This doesn't matter if you use the suggestion, but note that when iterating over the indices of an Array `dataInRow`, it's usually clearer to write this as:
`for (let j = 0; j < dataInRow.length; ++j)` 
(You probably already know this, but just in case you don't, this initializes `j` to `0`, checks that `j` is less than `dataRow.length`, if so proceeds to the body of the loop, then updates by adding `1` to `j` (`++j` is shorthand for `j = j + 1`), checks that this new value of `j` is less than `dataRow.length` and if so proceeds to the body of the loop, updates `j` by adding `1` to get `2`, etc.)
However, in this case, using this form has the additional benefit of allowing you to making your code shorter since you only want to continue while `j < 3`.  So the entire loop would become:
```
for (let j = 0; j < 3; ++j) {
  temArray.push(dataInRow[j]);
}
```
In general, you want to try to avoid using `break` in a loop if you can because it can make your code more difficult to understand.